services/src/test/scala/cromwell/pipeline/service/AuthServiceTest.scala

```diff
@@ -175,18 +172,24 @@ class AuthServiceTest extends WordSpec with Matchers with MockFactory {
     }
 
     "userIsActive" should {
-      "return None if a user is active" in {
+      "return None if a user is active" taggedAs Service in new AuthServiceTestContext {
         authService.userIsActive(dummyUser) shouldBe None
       }
 
-      "throw the exception if a user isn't active" in {
+      "throw the exception if a user isn't active" taggedAs Service in new AuthServiceTestContext {
         authService.userIsActive(inactiveUser) shouldBe
           Some(InactiveUserException(AuthService.inactiveUserMessage))
       }
     }
   }
 
-  class RefreshTokenContext(lifetime: Long) {
+  class AuthServiceTestContext {
```

Comment:
typically this type of fixture-objects is traits (https://www.scalatest.org/user_guide/sharing_fixtures, Instantiating fixture-context objects), but here I see no problem with class

Reply:
I prefer traits for test fixtures too. I have used class here only because the tests already have a context based on the class.